cellarium/cas/settings/development.py

```diff
@@ -1 +1,2 @@
-CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api-dev.cellarium.ai"
+# CELLARIUM_CLOUD_BACKEND_URL = "https://cellarium-cloud-api-dev.cellarium.ai"
+CELLARIUM_CLOUD_BACKEND_URL = "http://localhost:8000"
```

Comment:
Did you mean to leave this?

Reply:
Oh yup, you're right.  I keep forgetting to switch that back.  I'll fix it